pkg/es/wrapper/wrapper_nolint.go

```diff
@@ -16,9 +16,6 @@ package eswrapper
 
 import "github.com/jaegertracing/jaeger/pkg/es"
 
-// Some of the functions of elastic.BulkIndexRequest violate golint rules,
-// e.g. Id() should be ID() and BodyJson() should be BodyJSON().
-
```

Comment:
this feels like the relevant comment that explains the need for this file.

Reply:
please restore the commeng

Reply:
I did it already

Reply:
it still shows as deleted in the latest version of PR

Reply:
my bad. Updated now